### PR TITLE
Update deployment Info in Getting-Started Guide 

### PR DESCRIPTION
**Pull request by menuka94**

### CODE REVIEW

File: examples/getting-started/README.md [hunk 1]
```diff
@@ -142,22 +115,82 @@ services:
...
+and run the `docker-compose.yml` by executing `docker-compose up -d`. 
+
+Now you have successfully deployed 'order-process-backend' and the 'composite-service' services in Docker. 
+(Execute `docker container ls` to see the running containers).
+
+Head over to the [Testing](#Testing) section to test your deployment.
+
+### Local Deployment
+
+Even though the Micro Integrator integrator is specifically designed to be container-native, it is possible for you to
```
Comment (a5anka) [inline examples/getting-started/README.md]:
```suggestion
Even though the Micro Integrator integrator is specifically designed to be container native, it is possible for you to 
```

File: examples/getting-started/README.md [hunk 2]
```diff
@@ -142,22 +115,82 @@ services:
...
 Please follow all the steps in the section [Create the order processing back-end](#1-create-the-order-processing-back-end) and
 [Create the composite service to invoke the back-end.](#2-create-the-composite-service-to-invoke-the-back-end) if you have not already done. In step 3 of
 [Create the composite service to invoke the back-end](#2-create-the-composite-service-to-invoke-the-back-end), you need
-to replace 'order-process-backend' endpoint address in [forwardOrderApi.xml](https://github.com/wso2/micro-integrator/blob/master/examples/getting-started/composite-service/composite-service/src/main/synapse-config/api/forwardOrderApi.xml) file as follows because in docker containers each container has their own localhost.
+to replace 'order-process-backend' endpoint address in 
+[forwardOrderApi.xml](https://github.com/wso2/micro-integrator/blob/master/examples/getting-started/composite-service/composite-service/src/main/synapse-config/api/forwardOrderApi.xml) 
+file as follows because in docker containers each container has its own localhost.
 ```xml
 <endpoint>
     <address uri="http://order-process-be-service:8290/order"/>
```
Comment (a5anka) [inline examples/getting-started/README.md]: Do we need to change the backend URL for k8s. Isn't it same for both Docker and k8s?

Comment (menuka94) [reply examples/getting-started/README.md]: If a user changes it to `localhost` in the local deployment, then they need to change it back for the K8s deployment.

File: examples/getting-started/README.md (hunk 1 above)
Comment (menuka94) [reply examples/getting-started/README.md]: Fixed.

File: examples/getting-started/README.md (hunk 2 above)
Comment (a5anka) [reply examples/getting-started/README.md]: :+1:

**Pull request merged**